ethclient/ethclient_test.go

```diff
@@ -598,8 +598,9 @@ func testAtFunctions(t *testing.T, client *rpc.Client) {
 	if err != nil {
 		t.Fatalf("unexpected error: %v", err)
 	}
-	if pending != 1 {
-		t.Fatalf("unexpected pending, wanted 1 got: %v", pending)
+	// After nerfing the pending block, pending transaction count will be 0
+	if pending != 0 {
```

Comment:
Why is the pending nonce changed here? I thought that's fetched through txpool.

Reply:
This is not the pending nonce, but the pending transaction count. The pending nonce does not change because its fetched through the txpool. The pending transaction count returns the number of transactions in a block. I don't think it is a useful metric at all for the pending block